src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md

```diff
@@ -0,0 +1,205 @@
+- The following [metrics]({% link v24.1/metrics.md %}) were added for improved observability into [disk bandwidth]({% link v24.1/common-issues-to-monitor.md %}#storage-and-disk-i-o):
+    - `storage.disk.read-max.bytespersecond`
+    - `storage.disk.write-max.bytespersecond` [#121553][#121553]
+- Added two new [cluster settings]({% link v24.1/cluster-settings.md %}), `storage.sstable.compression_algorithm_backup_storage` and `storage.sstable.compression_algorithm_backup_transport`, which in addition to the existing cluster setting `storage.sstable.compression_algorithm`, can be used to alter the compression algorithm used for various types of [SSTs]({% link v24.1/architecture/storage-layer.md %}#ssts). [#124245][#124245]
+
+<h3 id="v24-2-0-alpha-1-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- `SHOW CHANGEFEED JOB`, [`SHOW CHANGEFEED JOBS`]({% link v24.1/show-jobs.md %}#show-changefeed-jobs), and [`SHOW JOBS`]({% link v24.1/show-jobs.md %}) no longer expose user sensitive information like `client_key`. [#122529][#122529]
+- Added the new [option]({% link v24.1/create-changefeed.md %}#options) `encode_json_value_null_as_object` to JSON-formatted [changefeeds]({% link v24.1/change-data-capture-overview.md %}) that outputs `'null'::jsonb` as `{"__crdb_json_null__": true}` instead of `null`, to disambiguate between SQL-null and JSON-null. With this option enabled, if the literal value `{"__crdb_json_null__": true}` is present in a JSON value, it will have the same representation as JSON-null with this option enabled. If such a value is encountered in a changefeed, a (rate-limited) warning will be printed to the [`DEV` channel]({% link v24.1/logging.md %}). [#122848][#122848]
+- Added an error message for [changefeed]({% link v24.1/change-data-capture-overview.md %}) options and parameters that are not supported by the [Apache Pulsar sink]({% link v24.1/changefeed-sinks.md %}#apache-pulsar). [#124581][#124581]
+- [Scheduled changefeeds]({% link v24.1/create-schedule-for-changefeed.md %}) now pause after being [restored]({% link v24.1/restore.md %}) onto a different cluster, and after completion of [physical cluster replication]({% link v24.1/physical-cluster-replication-overview.md %}) to avoid inadvertent concurrent execution of the same schedule on multiple clusters. [#124631][#124631]
+- Added a new metric to facilitate the new [changefeed]({% link v24.1/change-data-capture-overview.md %}) billing strategy for usage in CockroachDB {{ site.data.products.cloud }} clusters. [#123145][#123145]
+- The `DEBUG_PAUSE_ON` option has been removed and replaced with the `restore.restore_after_failure` pause point to match other pause points used throughout CockroachDB. You can set this pause point by running: `SET CLUSTER SETTING jobs.debug.pausepoints = 'restore.after_restore_failure'`. [#125158][#125158]
```

Comment:
do we even have any docs on this "pause point" stuff?

is it related to https://www.cockroachlabs.com/docs/stable/pause-job ?